notebooks/random_forest_withdark.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>drop_duplicates operates on and returns a copy of the dataframe, so line 29 does nothing.  You either want</p><p>features = features.drop_duplicates, or features.drop_duplicates(..., inplace=True)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>